buggle.html

```diff
@@ -127,14 +130,22 @@ <h2 class="text-white">Buggle</h2>
           return shakenBuggleLetters;
         }
 
+	getRotClass(aSeed) {
+	  // One of 0, 90, 280, 270, all equally likely
+	  var deg = Math.abs(getSeededRandomNumber(0, 4, aSeed))*90; // uses hardcoded board size!
+	  return "rot" + deg;
+	}
+
         getSeededGameboard(aSeed) {
           var shakenBuggleLetters = this.getSeededShakenBuggleLetters(aSeed);
           var htmlString = '<table class="mx-auto">';
           for (var r = 0; r < 4; r++) {
             htmlString += "<tr>";
             for (var c = 0; c < 4; c++ ) {
-              htmlString += "<td class='w-25'>";
-              htmlString +=   "<button class='btn btn-info btn-lg rounded shadow buggleButton text-center w-100'>" + shakenBuggleLetters[(r*4) + c] + "</button>";
+	      htmlString += "<td class='w-25'>";
+	      htmlString +=   "<button class='btn btn-info btn-lg rounded shadow buggleButton text-center w-100'>";
+	      htmlString +=     "<div class='" + this.getRotClass((r+1)*4*(aSeed+1)*(c+1)) + "'>" + shakenBuggleLetters[(r*4) + c] + "</div>";

```

Comment:
I feel like I'm using your getSeededRandomNumber function wrong...  Everytime I call it in getRotClass it returns the same number so I change the seed every iteration